plugout_sdl.c

```diff
@@ -16,7 +16,6 @@
 #include "plugout.h"
 
 static const int PLAYBACK_MODE = 0;
-static const int NO_CHANGES_ALLOWED = 0;
 static const int UNPAUSE = 0;
 
 static const struct timespec SLEEP_INTERVAL = {
```

Comment:
style-wise, ALLCAPS should be for #defines

Reply:
I don't see why they can't be changed into #defines anyway, so... :)

Reply:
there goes the strong typing :)